superset/assets/src/dashboard/reducers/getInitialState.js

```diff
@@ -148,7 +148,18 @@ export default function (bootstrapData) {
   const sliceIds = new Set();
   const modalSliceIds = new Set();
 
+  var oldToNewSliceIdMap = dashboard.slices.reduce((map, slice) => {
+    map[slice.form_data.remote_id] = slice.slice_id;
```

Comment:
Can we test below use case to confirm if this works as intended.

I exported a dashboard from cluster A having pub-sub slice A with id  :3

I exported a dashboard from cluster B having pub-sub slice B with id 3

It should work correctly if I import both of this